### ISSUE TITLE
The player hangs after a song starts.

### ISSUE THREAD

**Issue by eliminyro** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2828]
**OS:**
Arch Linus 2017.10.01
Plasma 5.11.0

**Issue Descriptions:**
The player hangs after a song starts.
I suspect, it might be related to the fact that something is off with notifications: when launching the program via terminal, it shows warnings like this:
`(gpmdp:1923): libnotify-WARNING **: Failed to connect to proxy`

I could not find any way to disable notifications of the application to verify it, however, when the player reaches around a 50% of a song, the interface becomes available until the next song comes up.
During the period when the interface works as expected, no warnings in the terminal appear.

Stuff I tried:
1. Removed folders:
  -  .config/Google Play Music Desktop Player
  -  .cache/Google Play Music Desktop Player
2. Launched it with --disable-gpu parameter;
3. Rebooted the PC.
Same issue.

**Steps to Reproduce:**
1. Open player.
2. Play a song.

As a result, the interface hangs for some period of time and is not manageable.

Here's the debug: [debug_info.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/1390879/debug_info.zip)

**Comment by MarshallOfSound** [comment]
You can disable notifications inside Google Play Music Settings.  You'll find the "Settings" tab on the left menu, just disable desktop notifications and you should be good to go 👍

**Comment by BrianAllred** [comment]
It looks like either the DE you're using doesn't provide the notification
bus that Electron is using or you don't have it installed. What DE do you
use? And can you try installing Gnome and seeing that works?

**Comment by eliminyro** [comment]
@BrianAllred DE is KDE 5, I specified plasma version in the first post.
It used to work just fine, I don't know what has changed.

I disabled notifications as @MarshallOfSound suggested. I don't know how that slipped out of my sight in the first place. 
I don't really need them on my home PC, and it seems to solve my issue.
However, I can provide more information, if it helps to eliminate the cause.

Electron package is installed, but considered an orphan, so, most likely, it is not being used.
